public/keymaps/a/annepro2_c15_default.json

```
@@ -0,0 +1,29 @@
+{
+  "keyboard": "annepro2/c15",
+  "keymap": "default",
+  "version": 1,
```

I use a `commit` field that is not consumed by the API to track when a Configurator keymap needs to be updated.

```suggestion
  "commit": "dc67fd9b87b0e86cea8d176d8b9bbce3e8a8676d",
```